contracts/airdrops/TokenDrop.sol

```diff
@@ -4,85 +4,63 @@ pragma solidity ^0.8.14;
 import "@rari-capital/solmate/src/tokens/ERC20.sol";
 import "@rari-capital/solmate/src/tokens/ERC721.sol";
 import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
-import "@openzeppelin/contracts/access/Ownable.sol";
-import "../interfaces/AirdropInfo.sol";
 import "../interfaces/AirdropMerkleProof.sol";
 import "../interfaces/IAirBroFactory.sol";
 
 /// @title Airdrops new ERC20 tokens for airdrop recipients
-contract TokenDrop is ERC20, AirdropInfo, AirdropMerkleProof, Ownable {
+contract TokenDrop is ERC20, AirdropMerkleProof {
```

Comment:
For new ERC20, I'd use https://docs.openzeppelin.com/contracts/4.x/api/token/erc20#ERC20Permit
as they have gas-less approvals with EIP712 signatures

Reply:
Just gonna leave some useful links for later reference
- [ERC20Permin on OpenZeppelin](https://docs.openzeppelin.com/contracts/4.x/api/token/erc20#ERC20Permit-nonces-address-)
- [EIP-2612](https://eips.ethereum.org/EIPS/eip-2612)